Dizzy/Web Service/FirebaseWebService.swift

```diff
@@ -33,12 +33,18 @@ final class FirebaseWebService: WebServiceType {
                                                 completion: @escaping (Result<Response>) -> Void) {
         databaseReference.child(resource.path).observe(DataEventType.value) { snapshot in
             do {
-                guard let json = snapshot.value as? [String: Any] else {
+            
+                guard let jsonsArray = snapshot.value as? [String: Any] else {
                     completion(Result<Response>.failure(WebServiceError.noData))
                     return
                 }
                 
-                let data = try JSONSerialization.data(withJSONObject: json, options: .prettyPrinted)
+                var jsonToParse = [[String: Any]]()
+                for (_,value) in jsonsArray {
+                    jsonToParse.append(value as! [String : Any])
+                }
+                
+                let data = try JSONSerialization.data(withJSONObject: jsonToParse, options: .prettyPrinted)
                 let parsedData = try resource.parse(data: data)
```

Comment:
i think it's better if you extract the those 2 lines in to separate function called `getParsedData(from json:) -> Data` or something like this

Reply:
I think that it's overrated...